content/1_docs/3_reference/7_plugins/3_components/0_file-urls/component.txt

```diff
@@ -15,20 +19,3 @@ Kirby::plugin('my/fileUrls', [
 	]
 ]);
 ```
-
-### Params
```

Comment:
I assume these will be replaced with stuff from the source code?

Reply:
Yes